src/pages/STG_PanelHouseholds.page

```diff
@@ -106,228 +106,182 @@
-		                            <apex:selectOptions value="{!listSOInformalGreetingFormats}"/>
-		                    </apex:selectList>
-                            <apex:outputLabel value="{!$Label.stgLabelOtherInformalGreetingFormat}" styleClass="sr-only" for="txtFormatIG" />
-		                    <apex:inputText value="{!stgService.stgHN.Informal_Greeting_Format__c}" styleClass="form-control txtOther" 
-		                        onchange="setNameSpec(this, null, '{!$Label.stgLabelExamplesIGFormat}');" 
-		                        onfocus="setNameSpec(this, null, '{!$Label.stgLabelExamplesIGFormat}');" 
-		                        rendered="{!isEditMode}" id="txtFormatIG" style="display:none;"/>
-		                </apex:outputPanel>
-                    </div>
-                    <div class="col-sm-offset-4 col-sm-8 help-block">
-                        <apex:outputText value="{!$Label.stgHelpIGFormat}" />
-                    </div>
+        <div class="slds-section-title--divider">{!$Label.stgLabelHHNaming}</div>
+        <div class="slds-form--horizontal slds-m-around--large">
+            <div class="slds-form-element">
+                <apex:outputLabel value="{!$ObjectType.npo02__Households_Settings__c.Fields.npo02__Advanced_Household_Naming__c.Label}" for="cbxAHN" styleClass="slds-form-element__label" />
+                <div class="slds-form-element__control">
+                    <apex:inputcheckbox value="{!stgService.stgHH.npo02__Advanced_Household_Naming__c}" disabled="{!isReadOnlyMode}" id="cbxAHN" styleClass="slds-checkbox"/>
+                    <apex:outputText styleClass="slds-form-element__help" value="{!$Label.stgHelpHHNaming}"/>
                 </div>
-
-                <div class="form-group">
-                    <apex:outputLabel value="{!$ObjectType.Household_Naming_Settings__c.Fields.Name_Connector__c.Label}" for="txtAnd" styleClass="col-sm-4 control-label"  />
-                    <div class="col-sm-8 form-control-column">
-                        <apex:outputText value="{!stgService.stgHN.Name_Connector__c}" rendered="{!isReadOnlyMode}" />
-                        <apex:inputText value="{!stgService.stgHN.Name_Connector__c}" id="txtAnd" onchange="setExamples();" rendered="{!isEditMode}" styleClass="form-control" />
-                    </div>
-                    <div class="col-sm-offset-4 col-sm-8 help-block">
-                        <apex:outputText value="{!$Label.stgHelpNameConnector}" />
-                    </div>
+            </div>
+            <div class="slds-form-element">
+                <apex:outputLabel value="{!$ObjectType.Household_Naming_Settings__c.Fields.Household_Name_Format__c.Label}" for="slstrFormatHH" styleClass="slds-form-element__label" />
+                <div class="slds-form-element__control">
+                    <apex:outputText value="{!stgService.stgHN.Household_Name_Format__c}" rendered="{!isReadOnlyMode}" styleClass="slds-form-element__static" />
```

Comment:
The other textbox now appears too narrow.  it used to be as wide as all the other controls.  same goes for the two remaining other textboxes.